detection_rules/packaging.py

```diff
@@ -447,3 +464,38 @@ def generate_xslx(self, path):
     def bump_versions(self, save_changes=False, current_versions=None):
         """Bump the versions of all production rules included in a release and optionally save changes."""
         return manage_versions(self.rules, current_versions=current_versions, save_changes=save_changes)
+
+    def create_bulk_index_body(self) -> Tuple[Ndjson, Ndjson]:
+        """Create a body to bulk index into a stack."""
+        package_hash = self.get_package_hash(verbose=False)
+        now = datetime.datetime.isoformat(datetime.datetime.utcnow())
+        create = {'create': {'_index': f'rules-repo-{self.name}-{package_hash}'}}
+
+        # first doc is summary stats
+        summary_doc = {
+            'group_hash': package_hash,
+            'package_version': self.name,
+            'rule_count': len(self.rules),
+            'rule_ids': [],
+            'rule_names': [],
+            'rule_hashes': [],
+            'source': 'repo',
+            'details': {'datetime_uploaded': now}
+        }
+        bulk_upload_docs = Ndjson([create, summary_doc])
+        importable_rules_docs = Ndjson()
+
+        for rule in self.rules:
+            summary_doc['rule_ids'].append(rule.id)
+            summary_doc['rule_names'].append(rule.name)
+            summary_doc['rule_hashes'].append(rule.get_hash())
+            status = 'new' if rule.id in self.new_rules_ids else 'modified' if rule.id in self.changed_rule_ids \
```

Comment:
since this has two ifs together, I think an if/elif/else over more lines is better and more readable